### ISSUE TITLE
[Feature Request] Support for "file:///" URLs in "Quick Links" widget

### ISSUE THREAD

**Issue by lifeadventurer** [joelshepherd/tabliss#706]
Currently, the widget only supports `https://` URLs, which limits its usability when working with local files. I often need to reference local documentation, configuration files, and other resources stored on my machine. Allowing `file:///` URLs would greatly enhance the flexibility and usefulness of the widget.

**Comment by Emstar** [comment]
For that matter, browser config pages like `about:processes` do not work as Quick Links either. My expectation was that anything that could be entered as an URL would work.

**Comment by BookCatKid** [comment]
> For that matter, browser config pages like `about:processes` do not work as Quick Links either. My expectation was that anything that could be entered as an URL would work.

 I agree with that expectation, but unfortunately, Firefox has strict security restrictions that prevent extensions from redirecting to privileged URLs - this is a browser limitation that can't be worked around.

_I am not a maintainer of this project, but I do have my own fork [here](https://github.com/BookCatKid/tabliss-maintained) that has a fix for this issue on chromium coming in a release very soon._

**Comment by Emstar** [comment]
Thanks for explaining that, I have to admit the limitation makes sense.
